bin/reth/src/node/mod.rs

```diff
@@ -56,6 +56,7 @@ impl Command {
             init_genesis(db.clone(), serde_json::from_str(MAINNET_GENESIS).unwrap())?;
 
         info!("Connecting to p2p");
+        // ANCHOR: snippet-execute
```

Comment:
is this how we can link to code from markdown?

Reply:
Yup, this way the `{{#template ... }}` macros can just pull in the block of code surrounded by the `ANCHOR` comments